logstash-core/spec/logstash/json_spec.rb

```diff
@@ -56,6 +56,20 @@
         expect(LogStash::Json.dump(array)).to eql(json_array)
       end
 
+      context "pretty print" do
+
+        let(:hash) { { "foo" => "bar", :zoo => 2 } }
+
+        it "should serialize with pretty print" do
+          pprint_json = LogStash::Json.dump(hash, :pretty => true)
+          expect(pprint_json).to include("\n")
```

Comment:
I think I would make it stricter with comparing the results with the actual generated string, But I guess this work too?
```
irb(main):001:0> JrJackson::Base.generate({ "foo" => "bar", :zoo => 2 }, :pretty => true)
=> "{\n  \"foo\" : \"bar\",\n  \"zoo\" : 2\n}"
irb(main):002:0> JrJackson::Base.generate({ "foo" => "bar", :zoo => 2 }, :pretty => false)
=> "{\"foo\":\"bar\",\"zoo\":2}"